crates/proto/src/version/mod.rs

```diff
@@ -5,3 +5,4 @@ pub mod v686;
 pub mod v712;
 pub mod v729;
 pub mod v748;
```

Comment:
are there any stable versions between `v766` and `v748`?

Reply:
Nope, v766 is the latest stable, for some reason they haven't pulled it into the proto docs yet though

Reply:
Okay good to know!